src/Senseval3.jl

```diff
@@ -0,0 +1,82 @@
+struct Senseval3{S}
+    filepaths::Vector{S}
+end
+
+function Senseval3(dirpath, ver=3.0)
+    @assert(isdir(dirpath), dirpath)
+
+    if "wordnet$ver" ∈ readdir(dirpath)
+        paths = joinpath.(dirpath,"wordnet$ver", readdir(joinpath(dirpath, "wordnet$ver")))
+    else
+        paths = joinpath.(dirpath, readdir(dirpath))
+    end
+    Senseval3(paths)
+end
+
+Senseval3() = Senseval3(datadep"Senseval 3")
+
+MultiResolutionIterators.levelname_map(::Type{Senseval3}) = [
+    :doc=>1, :contextfile=>1, :context=>1,
+    :sent=>2,
+    :word=>3, :token=>4,
```

Comment:
```suggestion
    :word=>3, :token=>3,
```

Is this mistake in SemCore also? Words and tokens are the same thing

Reply:
Yes. I got confused with that. They should be the same thing. I will make the changes on both.